pyproject.toml

```diff
@@ -30,7 +31,8 @@ APScheduler = "*"
 redis = "*"
 matplotlib = "*"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
+pyproject-flake8 = "5.0.4"
```

Comment:
[discussion]

This version constraint comes from that flake8 > 6.0.0 needs python 3.8.1+. Can we go python 3.8.1+ ?
https://github.com/pycqa/flake8/pull/1741

Reply:
@hirosassa We may proceed to 3.8.1+, but would prefer another PR.